app/views/report_templates/_diff.html.erb

```diff
@@ -0,0 +1,6 @@
+<div class="editor-container">
+  <%= text_area_tag "diffLines","" ,:size => "12x30", :class => "col-md-12 editor_source diffMode", :label=>_("Template diff"), :'data-file-name' => file_name%>
```

Comment:
Weird spacing:
` "diffLines","" ,:size `
vs.
` "diffLines", "", :size `
and also add space before `%>` please